lib/inc/class-wp-rest-coauthors-authorterms.php

```diff
@@ -238,7 +238,7 @@ public function get_items( $request ) {
 
 		if ( empty( $terms ) ) {
 			//Nothing was returned, that shouldn't happen unless a requested post doesn't have any guest-authors
-			return new WP_Error( 'rest_authors_get_term', __( 'No terms returned.' ), array( 'status' => 404 ) );
+			return new WP_Error( 'rest_authors_get_term', __( 'No terms returned for "'.$this->coauthor_taxonomy.'" taxonomy.'  ), array( 'status' => 404 ) );
```

Comment:
Since it's coming in as a diff, may be an extra space before closing `)` in   `taxonomy.'  )`